projects/js-packages/licensing/components/activation-screen-error/index.tsx

```diff
@@ -0,0 +1,53 @@
+import jetpackAnalytics from '@automattic/jetpack-analytics';
+import { Icon, warning, check } from '@wordpress/icons';
+import React, { FC, useEffect } from 'react';
```

Comment:
Nitpick, I think it is best practices to import types separately below the other imports like
`import type { FC } from 'react'`

Reply:
Yes true, fixed. 👍